cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -835,7 +835,7 @@ installBuiltExe
 installBuiltExe verbosity overwritePolicy
                 sourceDir exeName finalExeName
                 installdir InstallMethodSymlink = do
-  notice verbosity $ "Symlinking '" <> exeName <> "'"
+  notice verbosity $ "Symlinking '" <> exeName <> "' to '" <> installdir <> "'"
```

Comment:
I would rather print `installdir </> finalExeName` (i.e. `destination` in `installBuiltExe`). 

This is because one might link `foo` into `/some/dir/foo-1.0.0` if there is an executable executable suffix asked for.

Reply:
Should we also include the `finalExeName` when copying, or only when symlinking?

Reply:
Yes, by the same reasoning.

Reply:
Done!